plugins/discourse-presence/assets/javascripts/discourse/components/topic-presence-display.js.es6

```diff
@@ -15,8 +19,17 @@ export default Component.extend({
 
   shouldDisplay: gt("users.length", 0),
 
+  @observes("topic.id")
```

Comment:
we really want to avoid observers, could it work in didReceiveAttrs hook ?

Reply:
Yes, it does.

(I was hoping I could fix this without adding an extra `topicId` attribute, but I couldn't find a way. If you have any ideas please let me know.)